src/cli/similar.ts

```
@@ -0,0 +1,307 @@
+        threshold: parts[1] ? parseFloat(parts[1]) : 0.5
+      };
+    
+    case 'intersection':
+      return { strategy: 'intersection' };
+    
+    case 'union':
+      return { strategy: 'union' };
+    
+    case 'weighted':
+      // Parse weighted format: weighted:detector1=0.5,detector2=0.3
+      const weightings: Record<string, number> = {};
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)